tests/core/policies/test_memoization.py

```diff
@@ -422,10 +351,9 @@ def test_aug_pred_sensitive_to_intent_across_max_history_actions(self, max_histo
         )
```

Comment:
The asserted `action_listen` in the above line is something new to me -- @kedz do you know where it's coming from? I'm just curious because here it looks as if listening was the default action but it's the first time I've encountered this design decision...

Reply:
I think I should change the test because this actually not the default behavior but falling out of how the test checks for the action name.

in this case `prediction2.probabilities == [0.0, 0.0, ..., 0.0]` so the argmax of it is index 0 which is usually `action_listen`, but I don't know if we guarantee it to be so. I'll mod the test to assert that all probabilities are zero.

Reply:
Aaaah, I see! Alrighty, the changes you've made make a complete sense to me 🙂